phpBB/phpbb/cache/driver/file.php

```diff
@@ -584,4 +587,21 @@ function _write($filename, $data = null, $expires = 0, $query = '')
 
 		return $return_value;
 	}
+
+	/**
+	* 
+	* Change the cache path name for extensions
+	*
+	* {@inheritDoc}
+	* @param string $name extension filename
+	* @return string the corrected path of extension
+	*/
+	function extension_path($name)
```

Comment:
This has nothing to do with extensions, its a general issue.
Maybe `get_file_name()` ?